### ISSUE TITLE
Issue where the board shows with all the tiles grouped together at the top.

### ISSUE THREAD

**Issue by portaljumper007** [PirateGame/Pirate-Game-Frontend#45]

**Comment by Alex40144** [comment]
this is similar to the issue of baord being collapsed weirdly on a small screen

**Comment by portaljumper007** [comment]
> this is similar to the issue of baord being collapsed weirdly on a small screen

Yes, I think it may be something to do with the minimum pixel width of each square on the grid, if we break through that condition with a resize, it just sort of messes it all up.

**Comment by Alex40144** [comment]
if you revisit the page, it fixes itself.

some javascript issue. moving to low priority
